samtranslator/model/apigatewayv2.py

```diff
@@ -36,6 +36,25 @@ class ApiGatewayV2Stage(Resource):
     runtime_attrs = {"stage_name": lambda self: ref(self.logical_id)}
 
 
+class ApiGatewayV2DomainName(Resource):
+    resource_type = "AWS::ApiGatewayV2::DomainName"
+    property_types = {
+        "DomainName": PropertyType(True, is_str()),
+        "DomainNameConfigurations": PropertyType(False, list_of(is_type(dict))),
+        "Tags": PropertyType(False, list_of(is_type(dict))),
+    }
+
```

Comment:
I see ref values for `DomainName` [here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigatewayv2-domainname.html). Do you think adding `runtime_attrs` for `AWS::ApiGatewayV2::DomainName` will be useful?